bindings/wasm/src/did/wasm_document.rs

```diff
@@ -497,3 +660,57 @@ extern "C" {
   #[wasm_bindgen(typescript_type = "DIDUrl | string")]
   pub type UDIDUrlQuery;
 }
+
+#[wasm_bindgen]
+extern "C" {
+  #[wasm_bindgen(typescript_type = "DID[]")]
+  pub type DIDArray;
+}
```

Comment:
Nit: this should match current naming conventions.

```suggestion
#[wasm_bindgen]
extern "C" {
  #[wasm_bindgen(typescript_type = "DID[]")]
  pub type ArrayDID;
}
```